README.md

```diff
@@ -33,620 +33,677 @@ If you want to build the CLI using the source code, please, take into account th
+5aac0c0e348f190b3e001432   Basic_A0              768            1              20             5aabb7551de0240abb000060   5aabb7511de0240abb000005
+5aac0c10348f190b3e0014dd   Basic_A0              768            1              20             5aabb7551de0240abb000060   5aabb7511de0240abb000005
+5aac0c13348f190b3e001588   Basic_A0              768            1              20             5aabb7551de0240abb000061   5aabb7511de0240abb000005
+5aac0c15348f190b3e001633   Basic_A0              768            1              20             5aabb7551de0240abb000061   5aabb7511de0240abb000005
+5aac0c18348f190b3e0016de   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c1b348f190b3e001789   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c1d348f190b3e001834   Basic_A0              768            1              20             5aabb7551de0240abb000063   5aabb7511de0240abb000005
+5aac0c20348f190b3e0018df   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c23348f190b3e00198a   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c26348f190b3e001a35   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c29348f190b3e001ae0   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c2c348f190b3e001b8b   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c2f348f190b3e001c36   Basic_A0              768            1              20             5aabb7551de0240abb000060   5aabb7511de0240abb000005
+5aac0c32348f190b3e001ce1   Basic_A0              768            1              20             5aabb7551de0240abb000060   5aabb7511de0240abb000005
+5aac0c35348f190b3e001d8c   Basic_A0              768            1              20             5aabb7551de0240abb000061   5aabb7511de0240abb000005
+5aac0c38348f190b3e001e37   Basic_A0              768            1              20             5aabb7551de0240abb000061   5aabb7511de0240abb000005
+5aac0c3c348f190b3e001ee2   Basic_A0              768            1              20             5aabb7551de0240abb000060   5aabb7511de0240abb000005
+5aac0c3f348f190b3e001f8d   Basic_A0              768            1              20             5aabb7551de0240abb000060   5aabb7511de0240abb000005
+5aac0c42348f190b3e002038   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+5aac0c45348f190b3e0020e3   Basic_A0              768            1              20             5aabb7551de0240abb000062   5aabb7511de0240abb000005
+```
```

Comment:
Again, we should consider showing the name and id of the cloud provider and location to simplify this workflow, by making a cloud provider listing API request, a location listing API request and then mapping them to the results of the current server plan listing.

Reply:
created PR #74